src/components/Explore/Feed.tsx

```diff
@@ -70,13 +67,23 @@ const Feed: FC<Props> = ({ focus, feedType = PublicationSortCriteria.CuratedProf
       next={loadMore}
       loader={<InfiniteLoader />}
     >
-      <Card className="divide-y-[1px] dark:divide-gray-700/80">
-        {publications?.map((publication, index: number) => (
-          <SinglePublication
-            key={`${publication.id}_${index}`}
-            publication={publication as LensterPublication}
-          />
-        ))}
+      <Card>
+        <Virtuoso
+          useWindowScroll
+          className="virtual-list"
+          totalCount={publications?.length}
+          components={{ Footer: () => <PublicationsShimmer inVirtualList /> }}
+          itemContent={(index) => {
+            const publication = publications?.[index] as LensterPublication;
```

Comment:
Why are we type casting again in line number 82?

Reply:
My wrong fixed 😊